src/ncp/spinel.h

```diff
@@ -1054,6 +1054,7 @@ enum
     SPINEL_CAP_CHILD_SUPERVISION       = (SPINEL_CAP_OPENTHREAD__BEGIN + 8),
     SPINEL_CAP_POSIX_APP               = (SPINEL_CAP_OPENTHREAD__BEGIN + 9),
     SPINEL_CAP_SLAAC                   = (SPINEL_CAP_OPENTHREAD__BEGIN + 10),
+    SPINEL_CAP_COEX_METRICS            = (SPINEL_CAP_OPENTHREAD__BEGIN + 11),
```

Comment:
I don't think it's necessary to add a new capability. If this features is not enabled, getting this property will responds not found error, which is already enough to indicate this features is not supported. This way, we save code size.

Reply:
Great. I have removed the capability property.